app/models/subnet.rb

```diff
@@ -363,6 +365,13 @@ def dns_servers
     [dns_primary, dns_secondary].select(&:present?)
   end
 
+  def masked_subnet_address
+    IPAddr.new(network).mask(mask)
+  rescue StandardError => e
+    logger.info "Network address #{network} cannot be masked: #{e}"
+    network
```

Comment:
this returns an `IPAddr` in the case it could be masked, and a `String` otherwise, which feels odd?

Reply:
`Net::Validations.normalize_network_address` always returns a string and is otherwise identical to this function?